tests/end2end/test_invocations.py

```diff
@@ -103,6 +103,23 @@ def test_ascii_locale(request, server, tmpdir, quteproc_new):
 
 
 @pytest.mark.linux
+def test_open_with_ascii_locale(request, server, tmpdir, quteproc_new):
+    """Test opening non-ascii URL with LC_ALL=C set.
+
+    https://github.com/qutebrowser/qutebrowser/issues/1450
+    """
+    args = ['--temp-basedir'] + _base_args(request.config)
```

Comment:
Might want to also pass a file to open here on the commandline, as that seemed to trigger a slightly different  bug?

Reply:
It doesn't crash, but just shows URL_NOT_FOUND, which I think would be correct. Regardless, I added a test for it in case that breaks in the future.